doc/build-fastd-android.sh

```diff
@@ -105,7 +109,7 @@ if [ ! -d "pkgconfig" ]; then
 fi
 
 # detect HomeBrew installed bison for OS X
-HOMEBREW_BISON_PATH=`find /usr/local/Cellar/bison -name bin`
+HOMEBREW_BISON_PATH=`find /usr/local/Cellar/bison -name bin | tail -n 1`
```

Comment:
How about using `/usr/local/opt/bison/bin` which is always linked to the latest Homebrew bison installation instead of the weird find?